### PR TITLE
document linkerd and namerd admin endpoints

### LINKED ISSUE #1012: document admin endpoints
Currently things like `/admin/tracing`, `/admin/shutdown` are not documented in config.md

### PR DESCRIPTION
**Pull request by siggy**
fixes #1012

screen shot 2017-02-01 at 6 38 37 pm

### CODE REVIEW

File: linkerd/docs/config.md [hunk 1]
```diff
@@ -90,6 +90,56 @@ Key | Default Value | Description
...
+`/admin/threads` | capture the current stacktraces, set `Accept: application/json` to force json
+`/admin/threads.json` | identical to `admin/threads`
+`/admin/tracing` | enable (`/admin/tracing?enable=true`) or disable tracing (`/admin/tracing?disable=true`)
+`/config.json` | current linkerd configuration, this should match your config file
+
+Endpoints only available in linkerd:
+
+Endpoint | Description
+-------- | -----------
+`/bound-names.json` | list of all known bound names from configured namers
```
Comment (klingerf) [inline linkerd/docs/config.md]: TIOLI it might be worth mentioning here that this only works for the k8s and fs namers.

Comment (adleong) [inline linkerd/docs/config.md]: or maybe, more generally, that it only works for namers that support this enumeration.

**Comment by kunalpathak14** [comment]
⭐️ thanks this is really helpful.  @siggy my delegator.json curl command  is not dynamically changes the endpoints after i am flipping namer endpoint and need to kick start linkerd service to get the correct IPs. any inputs?

**Comment by siggy** [comment]
@kunalpathak14 delegator.json is a read-only endpoint, not sure if that's what you mean. recommend asking with a bit more detail on https://discourse.linkerd.io/ or in linkerd slack.

**Pull request merged**